_rules/html-page-title-2779a5.md

```diff
@@ -39,23 +39,17 @@ htmlHintIgnore:
 
 The root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), if it is an `html` element.
```

Comment:
We might want to update that to use https://act-rules.github.io/glossary/#web-page-html instead. This would remove the need for "if it is an `html` element" since our definition is only about HTML.

This is most likely topic for another PR, however. But may be related to https://github.com/act-rules/act-rules.github.io/pull/1340/files#r438130594

Reply:
Go for it.